### ISSUE TITLE
Newtonsoft.Json.JsonSerializationException: Error Converting Value "Disabled"

### ISSUE THREAD

**Issue by ooples** [alpacahq/alpaca-trade-api-csharp#299]
Exception Type:
Newtonsoft.Json.JsonSerializationException

```cs
Exception Message:
Error converting value "DISABLED" to type 'Alpaca.Markets.AccountStatus'. Path 'status', line 1, position 96.

Exception StackTrace:
   at Newtonsoft.Json.Converters.StringEnumConverter.ReadJson(JsonReader reader, Type objectType, Object existingValue, JsonSerializer serializer)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.DeserializeConvertable(JsonConverter converter, JsonReader reader, Type objectType, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.SetPropertyValue(JsonProperty property, JsonConverter propertyConverter, JsonContainerContract containerContract, JsonProperty containerProperty, JsonReader reader, Object target)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(Object newObject, JsonReader reader, JsonObjectContract contract, JsonProperty member, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonSerializer.Deserialize(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonSerializer.Deserialize[T](JsonReader reader)
   at Alpaca.Markets.HttpResponseMethodExtensions.DeserializeAsync[TApi,TJson](HttpResponseMessage response)
   at Alpaca.Markets.HttpClientExtensions.callAndDeserializeAsync[TApi,TJson](HttpClient httpClient, HttpRequestMessage request, CancellationToken cancellationToken, IThrottler throttler)
   at Alpaca.Markets.HttpClientExtensions.callAndDeserializeAsync[TApi,TJson](HttpClient httpClient, HttpMethod method, Uri endpointUri, CancellationToken cancellationToken, IThrottler throttler)
   at OoplesFinanceAdminClient.AlpacaApi.PrintAPIDetails() in D:\Code Projects\OoplesFinanceAdminClient\AlpacaApi.cs:line 1233

Exception Target Site:
ReadJson

Exception Source:
Newtonsoft.Json

Requested value 'DISABLED' was not found.

   at Newtonsoft.Json.Utilities.EnumUtils.ParseEnum(Type enumType, NamingStrategy namingStrategy, String value, Boolean disallowNumber)
   at Newtonsoft.Json.Converters.StringEnumConverter.ReadJson(JsonReader reader, Type objectType, Object existingValue, JsonSerializer serializer)
```

I am getting this error on the latest version when I try to check my account status.

**Comment by OlegRa** [comment]
@ooples This account status is not documented [here](https://alpaca.markets/docs/api-documentation/api-v2/account/) but I'll add this value into enum in the next release.

**Comment by OlegRa** [comment]
@ooples This issue fixed in the latest [5.0.1-alpha5](../releases/tag/sdk-5.0.1-alpha5) release. Please, test and close.

**Comment by ooples** [comment]
It looks like the problem has been fixed
